[RHACS] ROX-17476: Restructuring the roxctl guide (#62722)
openshift/openshift-docs: modules/generating-sensor-deployment-files.adoc

// Module included in the following assemblies:
//
// * cli/managing-secured-clusters.adoc
:_content-type: PROCEDURE
[id="generating-sensor-deployment-files_{context}"]
= Generating Sensor deployment files

[discrete]
== Kubernetes

.Procedure

* Generate the required sensor configuration for your Kubernetes cluster and associate it with your Central instance by running the following command:
+
[source,terminal,subs="+quotes"]
----

> $ roxctl -e "$ROX_CENTRAL_ADDRESS" sensor generate k8s --name _<cluster_name>_ --central "$ROX_CENTRAL_ADDRESS"

Reviewer: Same notion w.r.t. the `-e` flag.

----

[discrete]
== {ocp}

.Procedure

* Generate the required sensor configuration for your OpenShift cluster and associate it with your Central instance by running the following command:
+
[source,terminal,subs="+quotes"]
----

> $ roxctl -e "$ROX_CENTRAL_ADDRESS" sensor generate openshift --openshift-version _<ocp_version>_ --name _<cluster_name>_ --central "$ROX_CENTRAL_ADDRESS" <1>

Reviewer: Same notion w.r.t. the `-e` flag.

// Module included in the following assemblies:
//
// * cli/managing-secured-clusters.adoc
:_content-type: PROCEDURE
[id="generating-sensor-deployment-files_{context}"]
= Generating Sensor deployment files

[discrete]
== Kubernetes

.Procedure

* Generate the required sensor configuration for your Kubernetes cluster and associate it with your ROX Central instance:
+
[source,terminal,subs="+quotes"]
----
$ roxctl -e "$ROX_CENTRAL_ADDRESS" sensor generate k8s --name _<cluster_name>_ --central "$ROX_CENTRAL_ADDRESS"
----

> 

Reviewer 2: remove empty line

[discrete]
== {ocp}

.Procedure

> * Generate the required sensor configuration for your OpenShift cluster and associate it with your ROX Central instance:

Reviewer 2: ```suggestion
* Generate the required sensor configuration for your {ocp} cluster and associate it with your Central instance:
```

+
[source,terminal,subs="+quotes"]
----
$ roxctl -e "$ROX_CENTRAL_ADDRESS" sensor generate openshift --openshift-version _<ocp_version>_ --name _<cluster_name>_ --central "$ROX_CENTRAL_ADDRESS" <1>
----
+

> <1> For the `--openshift-version` option specify the major {ocp} version number for your cluster. For example, specify `3` for {ocp} version `3.x` and specify `4` for {ocp} version `4.x`.

Reviewer 2: ```suggestion
<1> For the `--openshift-version` option, specify the major {ocp} version number for your cluster. For example, specify `3` for {ocp} version `3.x` and specify `4` for {ocp} version `4.x`.
```

+
Read the `--help` output to see other options that you might need to use depending on your system architecture.
+
Verify that the endpoint you provide for `--central` can be reached from the cluster where you are deploying {product-title} services.
+
[IMPORTANT]
====

> If you are using a non-gRPC capable load balancer, such as HAProxy, AWS Application Load Balancer (ALB), or AWS Elastic Load Balancing (ELB):

Reviewer 2: ```suggestion
If you are using a non-gRPC capable load balancer, such as HAProxy, AWS Application Load Balancer (ALB), or AWS Elastic Load Balancing (ELB), follow these guidelines:
```
